### PR TITLE
Add GCP deploying extension nodes chapter

### PR DESCRIPTION
**Pull request by ianf77**
New text

### CODE REVIEW

File: stories/topics/con-gcp-pull-deploy-container-image.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+[id="con-gcp-pull-deploy-container-image"]
+
+= Pulling the ansible-on-clouds-ops container image
+
+Pull the Docker image for the Ansible on Clouds operational container with the same tag as the version you are deploying to.
+
+[NOTE]
+====
+Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com.
```
Comment (leena-jawale) [inline stories/topics/con-gcp-pull-deploy-container-image.adoc:9]: it should be `registry.redhat.io` in both places. Sorry forgot to uodate this in hackmd

**Review by zkayyali812 — APPROVED** [review]

File: stories/topics/ref-gcp-decide-offer-type.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+The following table lists the offer types and the corresponding GCP instance types. 
+Depending on the workload needs, you can choose a more suitable offer type for the extension nodes.
+
+[options="header",width="60%",cols="^s,<"]
+|====
+| Offer Type (nodes) | GCP Instance Type
+| 100 | n2-standard-2
+| 200 | n2-standard-4
+| 400 | n2-standard-8
```
Comment (ariordan-redhat) [inline stories/topics/ref-gcp-decide-offer-type.adoc:13]: Can you left-justify the 1st column? This is how it renders in the build:
image

File: stories/topics/ref-gcp-permissions-to-manage-nodes.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[id="ref-gcp-permissions-to-manage-nodes_{context}"]
+
+= IAM minimum permissions
+
+Your GCP account must have the following _Identity and Access Management_ (IAM) permissions to successfully create and manage extension nodes on {PlatformNameShort}.
+
+Your GCP account must also be licensed to deploy Extension node offers for {AAPonGCP}.
```
Comment (ariordan-redhat) [inline stories/topics/ref-gcp-permissions-to-manage-nodes.adoc:7]:
```suggestion
Your GCP account must also be licensed to deploy extension node offers for {AAPonGCP}.
```
Does Extension need to be uppercase?

**Review by ariordan-redhat — COMMENTED** [review]
Just a couple of minor suggestions

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**